program/program/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.IO;
+
+namespace ConsoleApp
+{
+    class Auto
+    {
+        public int AmountOfWheels { get; set; }
+        public string TypeOfEngine { get; set; }
+        public string TypeOfAuto { get; set; }
+        public string Color { get; set; }
+
+    }
+
+    class TypeOfTheTransport
+    {
+        public string PassengerCar { get; set; }
+        public string Truck { get; set; }
+        public string Minivan { get; set; }
+        public string Motorcycle { get; set; }
+        public string ATV { get; set; }
+        public string MissleComplex { get; set; }
+
+    }
+
+    class TypeOfTheProblam
+    {
+        public string Scratch { get; set; }
```

Comment:
it's better to do with inheritance, Problem is base class and every type of problem is child